primevue-gen/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "primvue-gen",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "esbuild": "0.19.5",
+    "primevue": "3.40.1"
+  }
+}
```

Comment:
General comments about this file (`primevue-gen/package.json`) and also `package-lock.json`:
- Since we are pinning the versions of the packages directly in this file, I think we should get rid of the `package-lock.json` file by running `npm i --no-package-lock`. Also we should add `package-lock.json` to `.gitignore`.
- I don't know if you noticed, but the content of the `package-lock.json` file has internal URLs on it, probably because you have set a local mirror for NPM repositories. Thouse URLs would not work for anyone outside your company (Promgen is an open source project). Since other people may also want to use their own NPM mirrors, I think it's another reason for not generating (and commit) a `package-lock.json` file.

Reply:
This is not completely resolved:
- You ignored `package-lock.json` with 492ccde551746518465a0c7d0b929098ca871350.
- You removed `package-lock.json` with bfb968b0b33a3f0803f338c0c23ba2557147c604.

But the `npm` command stills needs to be addressed to avoid generating that file: `npm i --no-package-lock`.

Reply:
Addressed the comment in [f4c9367](https://github.com/line/promgen/pull/456/commits/f4c9367e5793a87e9da76026f37268a149bceecd)